<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="capframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string>原始视频</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>810</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停/播放</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="deepframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string>跟踪检测</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>750</y>
      <width>80</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>显示检测</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>700</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="historycombox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1个小时前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2个小时前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3个小时前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4个小时前</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>今天15点</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>今天16点</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>今天17点</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>今天18点</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="reseTrackerBut">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>重新检测</string>
    </property>
   </widget>
   <widget class="QToolButton" name="hikcambut">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时时监控</string>
    </property>
   </widget>
   <widget class="QToolButton" name="hikcamhisitory">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>监控录像</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>210</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearbut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>810</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清理所有进程</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoinfolabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="roiBut">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择区域</string>
    </property>
   </widget>
   <widget class="QToolButton" name="readVideoBut">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>读取记录</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showgoprovideo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>显示Gopro</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showGoproSound">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>gopro声音</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>121</width>
      <height>223</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="showprogressbar">
       <property name="text">
        <string>显示视频进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowRoiRect">
       <property name="text">
        <string>显示选择区域</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowMaskFrame">
       <property name="text">
        <string>显示检测mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSaveVideo">
       <property name="text">
        <string>保存识别视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saveFileName">
       <property name="text">
        <string>tiktok.mp4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="capStopCheck">
       <property name="text">
        <string>视频暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="deepStopCheck">
       <property name="text">
        <string>检测暂停</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="selectCamcombox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>790</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>212</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>231</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="waitDeepLenSlide">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="waitdeepnumtxt">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>内存数</string>
    </property>
   </widget>
   <widget class="QLabel" name="fpsnumtxt">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fps</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
